portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -73,6 +157,67 @@ definition {
 		}
 	}
 
+	@description = "LPS-122873 When there are some categories created for vocabulary1 (X axis), they will always have the same order during the user session"
+	@priority = "3"
+	test CategoriesWithSameOrderDuringASession {
+		property portal.acceptance = "true";
+
+		task ("Create a list of Audiences") {
+			for (var count : list "1,2,3") {
+				JSONCategory.addCategory(
+					groupName = "Global",
+					title = "S ${count}",
+					vocabularyName = "Audience");
+			}
+		}
+
+		task ("Create a list of Stages") {
+			for (var count : list "1,2,3") {
+				JSONCategory.addCategory(
+					groupName = "Global",
+					title = "test ${count}",
+					vocabularyName = "Stage");
+			}
+		}
+
+		task ("Create a Web Content") {
+			for (var count : list "1,2,3") {
+				JSONWebcontent.addWebContent(
+					content = "Webcontent Content",
+					groupName = "Test Site Name",
+					title = "Webcontent Title ${count}");
+			}
+		}
+
+		task ("Select of categories") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+			for (var count : list "1,2,3") {
+				WebContentNavigator.gotoEditCP(webContentTitle = "Webcontent Title ${count}");
+
+				for (var count : list "1,2,3") {
+					WebContent.selectCategory(
+						categoryName = "S ${count}",
+						vocabularyName = "Audience");
+				}
+
+				for (var count : list "1,2,3") {
+					WebContent.selectCategory(
+						categoryName = "test ${count}",
+						vocabularyName = "Stage");
+				}
+
+				PortletEntry.publish();
+			}
+		}
+		task ("Navigate to content Dashboard and View the order of categories") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+		}
```

Comment:
Here you say that it navigates and check the category order but you are just navigating. Where are the asserts?